docs/source/operations/howto/sso-oidc.rst

```diff
@@ -0,0 +1,76 @@
+
+Get your **provider URL**. Appending ``/.well-known/openid-configuration`` to this URL should yield a JSON description of the provider's capabilities, which Substra will use for much of the configuration (else, you can set endpoints by hand under ``oidc.provider.endpoints``.
+
+The provider will give you a **client id** and a **client secret**. Deploy them on the cluster in a secret:
+
+.. code-block:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: oidc-secret
+   stringData:
+     OIDC_RP_CLIENT_ID: "CLIENT_ID"
+     OIDC_RP_CLIENT_SECRET: "CLIENT_SECRET"
+
+
+Set up user creation
+====================
+
+When a user first signs on through OIDC, they are assigned an username and channel. 
+
+You must decide to which channel the OIDC users will be assigned with the ``oidc.users.channel``.
+
+The username is based on user email, but can be customized with the ``oidc.users.appendDomain`` option.
+
+A note on user validity
+-----------------------
+
+An issue is if a user gets their account disabled at the provider, but still has access to your Substra instance. You can use ``oidc.users.loginValidityDuration`` to mitigate this: accounts that have not logged in in this amount of time are disabled until the user signs on again. The API tokens associated with their account stop working as well, but will work again when they refresh their login.
+
+Since this is irritating to users, Substra will attempt to fetch user info in the background if possible, so the process is transparent to users. However this requires the provider to support offline access and refresh tokens -- not all do, and implementations vary.
+
+Automated login refresh is enabled by default under the option ``oidc.users.useRefreshToken``, but Substra will disable it and all back to the manual mode if it can't detect provider support.
+
+If you are using automated login refresh, you can set ``oidc.users.loginValidityDuration`` to low values to slightly increase security at a small cost in server load. Otherwise, it is a balance of security versus user convenience.
+
+Other settings
+==============
+
+If OIDC users will be using the Substra API (if they are data scientists running Pythong scripts, for instance), they'll need to generate API tokens on the web frontend and use those in their scripts.
+
+Having to generate new tokens all the time is a hindrance to the users' work, so you can increase their lifetime through ``config.EXPIRY_TOKEN_LIFETIME`` in the backend values.
+
+Final values
+============
+
+You can then set up OIDC in the backend values:
+
+.. code-block:: yaml
+
+   config:
+     EXPIRY_TOKEN_LIFETIME: "10080" # one week, in minutes
+   oidc:
+     enabled: true
+     clientSecretName: oidc-secret
+     provider:
+       url: "PROVIDER_URL"
+       displayName: Anything you like here :) It will be displayed to users.
+     users:
+       channel: CHANNEL_ID
+
+
+You'll still need to create admin accounts as normal.
```

Comment:
I would be in favor of stating the limitations of SSO at the beginning of the page. For me there are two points:
- `admin` users have to be created through another mean (perhaps we should create a command to change a user from `user` to `admin`)
- SSO users are always added to one channel